Tests/README.md

```diff
@@ -48,8 +56,9 @@ docker build --tag scs-compliance-check .
 docker run -it --env OS_CLOUD=CLOUDNAME -v ~/.config/openstack:/root/.config/openstack:ro scs-compliance-check
 ```
 
-The Docker entrypoint uses [scs-compatible.yaml](scs-compatible.yaml) on the `iaas` layer by default. You can use an alternative spec file
-by simply appending it to the above call, e.g.
+The Docker entrypoint uses [scs-compatible-iaas.yaml](scs-compatible-iaas.yaml)
```

Comment:
I'm afraid that this isn't true any more (not your fault). Could you update the Dockerfile as well?

Reply:
Never mind, I did it myself. But while testing the Docker build (which worked okay), I got this message: 
```console
DEPRECATED: The legacy builder is deprecated and will be removed in a future release.
            Install the buildx component to build images with BuildKit:
            https://docs.docker.com/go/buildx/
```
:(

Reply:
> I'm afraid that this isn't true any more (not your fault). Could you update the Dockerfile as well?

Good catch, didn't see that – thx!

> But while testing the Docker build (which worked okay), I got this message […]

This is a local issue and not related to the `Dockerfile`: you need to install the docker buildx plugin which will automatically used for `docker build`. On Debian/Ubuntu with the Docker APT repo just `apt install docker-buildx-plugin`.